ScienceJournal/UI/UserFlowViewController.swift

```diff
@@ -237,6 +226,10 @@ class UserFlowViewController: UIViewController, ExperimentsListViewControllerDel
 
     // Generate the default experiment if necessary.
     createDefaultExperimentIfNecessary()
+
+    if !devicePreferenceManager.hasAUserViewedOnboarding {
```

Comment:
```suggestion
    if !devicePreferenceManager.hasUserViewedOnboarding {
```
Any reason for the extra `A`?

Reply:
For consistency with existing properties.

Reply:
I see this is meant for a multi user environment

Reply:
Maybe because it's saved as a user default, so it's a per app setting, no matter what user is logged in.